src/app/containers/manuscript/manuscript-toc.tsx

```diff
@@ -14,10 +17,26 @@ export interface ManuscriptTOCProps {
 
 export const ManuscriptTOC: React.FC<ManuscriptTOCProps> = (props) => {
   const classes = useOutlinePanelStyles(props);
+  const dispatch = useDispatch();
   const { tocOpen, handleTocToggle } = props;
 
   const articleInfo = useSelector(getArticleInformation);
   const journalMeta = useSelector(getJournalMeta);
+  const manuscriptBodyTOC = useSelector(getManuscriptBodyTOC);
+  const toc: TableOfContent = [
+    { level: 1, title: 'Title', id: 'title' },
+    { level: 1, title: 'Abstract', id: 'abstract' },
+    { level: 1, title: 'Impact statement', id: 'impactStatement' },
+    ...manuscriptBodyTOC,
```

Comment:
There should be an entry here for 'main body' I think, which takes you to the start of the main body.